pci/src/vfio.rs

```diff
@@ -1491,6 +1534,15 @@ impl VfioPciDevice {
                         return Err(VfioPciError::MmapArea);
                     }
 
+                    if !is_page_size_aligned(area.size) || !is_page_size_aligned(area.offset) {
+                        warn!(
+                            "Could not mmap no page size aligned sparse area (offset = 0x{:x}, size = 0x{:x})",
+                            area.offset,
+                            area.size,
+                            );
+                        return Ok(());
```

Comment:
Is this fatal? Should it be treated as an error?

Reply:
I'm not sure if it's fatal. Sometims, it works without mmio map.